.travis.yml

```diff
@@ -162,6 +162,10 @@ cache:
         - $ALPAKA_CI_DOCKER_CACHE_DIR
 
 script:
+    - sudo apt-get -y --quiet --allow-unauthenticated --no-install-recommends install smem
+    - sudo smem
+    - sudo free -m -t
```

Comment:
I added some memory statistics here and there and stopped some services that are not necessary because the new variables in each kernel increased the build times and I tried to enable paralle compilation but this increased the memory consumption too much. So we are still doing sequential compilation but the memory statistics are now here ;-)